boa/vm/fork.py

```diff
@@ -236,14 +236,18 @@ def try_prefetch_state(self, msg: Message):
                 return
 
             # set account if we don't already have it
-            if self._get_account_helper(address) is None:
-                balance = to_int(v.get("balance", "0x"))
-                code = to_bytes(v.get("code", "0x"))
-                nonce = v.get("nonce", 0)  # already an int
+            account_helper = self._get_account_helper(address)
+            if account_helper is None:
```

Comment:
i think we should check `dontfetch` instead, it's designed so we don't have to peer into pyevm internals